src/main/resources/default/extensions/biz-tycho-menu/menu-jobs.html.pasta

```diff
@@ -7,7 +7,7 @@
             <t:dropdownItem labelKey="JobsController.jobs" url="/jobs"
                             permission="permission-execute-jobs"
                             framework="biz.jobs"/>
-            <t:dropdownItem labelKey="Process.plural" url="/ps"
+            <t:dropdownItem labelKey="Process.plural" url="/ps" permission="permission-system-protocols"
```

Comment:
Where is this permission granted?

Reply:
Currently it's only granted with the `flag-system-administrator` permission. Should I make it grantable by itself?